source/database/mssql/scripts/dbscripts/PSP_PIMS_S42_00/Test Data/38_PIMS_USER.sql

```diff
@@ -0,0 +1,276 @@
+DECLARE @devinPersonId bigint;
```

Comment:
So, this sets up your dev environment so that you can log into the application using your idir. By default you will have the System Admin role assigned and nothing else.